fendermint/vm/message/src/conv/from_fvm.rs

```diff
@@ -35,17 +35,15 @@ pub fn to_eth_tokens(amount: &TokenAmount) -> anyhow::Result<et::U256> {
     }
 }
 
-pub fn to_eth_address(addr: &Address, allow_masked: bool) -> anyhow::Result<Option<et::H160>> {
```

Comment:
Removing this `allow_masked`, which was added to differentiate a WASM actor addressed with a masked (Ethereum compatible) address and a "real" Ethereum contract.